realm/kotlin-extensions/src/androidTest/kotlin/io/realm/CoroutineTests.kt

```diff
@@ -134,6 +130,7 @@ class CoroutineTests {
         val countDownLatch = CountDownLatch(1)
 
         // Get results from the test thread
+        val realm = Realm.getInstance(configuration)
```

Comment:
Maybe wrap the underlying block in `use` instead of manually closing, because close will not be called if the `awaitOrFails` throws